lib/xnvme_be_spdk_dev.c

```diff
@@ -763,6 +763,16 @@ xnvme_be_spdk_state_init(struct xnvme_dev *dev)
 
 		XNVME_DEBUG("INFO: found dev->ident.uri: '%s' via cref_lookup()", dev->ident.uri);
 
+		err = spdk_nvme_ctrlr_process_admin_completions(state->ctrlr);
+		if (err < 0) {
+			XNVME_DEBUG("FAILED: spdk_nvme_ctrlr_process_admin_completions, err: %d",
+				    err);
+			if (_cref_deref(state->ctrlr)) {
```

Comment:
@safl @LiadOz This line needs to be changed to:
`if (_cref_deref(state->ctrlr, true)) {`